scripts/ci/setup.sh

```diff
@@ -6,6 +6,9 @@ ARGS+='this.dependencies_included = true;'
 # Process append to the version string if pre-release
 if [ "$TRAVIS_BRANCH" == "master" ]; then
     ARGS+="this.version += \"-alpha.$TRAVIS_BUILD_NUMBER\";"
+    # Run CI against latest master of submodules if this is master
+    git submodule update --remote api
```

Comment:
I don't think this does anything. Our submodules end up as un-attached git hashes, and so there is no update to perform:
```
scdf at SCdF in ~/Code/Medic/medic-webapp on master*
$ git submodule
 8106b25752ba60f481adf204177db91e53264e91 api (2.12.0-beta.1-24-g8106b25)
 6aff73fe39115a919ce961835e5bf0d88e307029 sentinel (2.12.0-beta.1-9-g6aff73f)
```

For this to work, you'd need to either:
 - Change the code to `cd api && git fetch && git checkout master` etc
 - Work out how we set up submodules that follow tips of branches, and also make sure that there is enough developer education (through either a quick workshop / post or through checklists) that this new system is understood for things like tags, where we have to be careful to not have this enabled)